pydis_site/templates/wiki/includes/breadcrumbs.html

```diff
@@ -0,0 +1,87 @@
+{% if urlpath and article and not user.is_anonymous %}
+  <section class="breadcrumb-section section">
+    <div class="container">
+      <nav class="breadcrumb is-pulled-left" aria-label="breadcrumbs">
+        <ul>
+          {% for ancestor in urlpath.cached_ancestors %}
+            <li>
+              <a href="{% url 'wiki:get' path=ancestor.path %}">{{ ancestor.article.current_revision.title|truncatechars:25 }}</a>
+            </li>
+          {% endfor %}
+
+          <li class="is-active">
+            <a href="{% url 'wiki:get' path=article.path %}">{{ article.current_revision.title|truncatechars:25 }}</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="dropdown is-pulled-right is-right">
+        <div class="dropdown-trigger">
+          <a aria-haspopup="true" aria-controls="sub-article-dropdown">
+            <span>Sub-Articles</span>
+            <span class="icon">
+              <i class="fas fa-angle-down"></i>
+            </span>
+          </a>
+        </div>
+        <div class="dropdown-menu" id="sub-article-dropdown" role="menu">
+          <div class="dropdown-content">
+            {% if children_slice %}
+              {% for child in children_slice %}
+                <a class="dropdown-item" href="{% url 'wiki:get' path=child.path %}">
+                  {{ child.article.current_revision.title }}
+                </a>
+              {% endfor %}
+
+              {% if children_slice_more %}
+                <a class="dropdown-item" href="{% url 'wiki:dir' path=urlpath.path %}">
+                  ...and more.
+                </a>
+              {% endif %}
+
+              <hr class="dropdown-divider">
+            {% endif %}
+
+            <a class="dropdown-item" href="{% url 'wiki:dir' path=urlpath.path %}">
+              Browse other articles
+            </a>
+          </div>
+        </div>
+      </div>
+
+      {% if request.user.is_authenticated %}
```

Comment:
i imagine not everybody should be able to create articles later? so we should make this an issue

Reply:
#201